dispatcher/templates/candidate_applied.jinja2

```diff
@@ -0,0 +1,82 @@
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="border: 1px solid #E0E4E8;"
+                    width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" style="padding: 56px 56px 28px 56px;" valign="top">
+                          <a style="color: #3498DB; text-decoration: none;" href="my.aegee.eu"
+                            target="_blank"><img style="width: 56px; height: 56px; border: 0;" alt="Logo"
+                              src="https://my.aegee.eu/images/logo.png">
+                          </a></td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 20px; color: #333;"><strong>A new candidature is submitted</strong></div>
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 16px; color: #333;">
+                            A new candidature to position
+                            <strong>{{ position["name"] }}</strong>
+                            is submitted by
+                            <strong>{{ candidate["first_name"] }} {{candidate["last_name"] }}</strong>.
+                          </div>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div>
+                            <a href="https://my.aegee.eu/statutory/{{ position["event_id"] }}/positions/all" style="background-color:#1468C5;border-radius:50px;color:#ffffff;display:inline-block;font-family: 'lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size:18px;font-weight: bold;line-height:40px;text-align:center;text-decoration:none;width:270px;-webkit-text-size-adjust:none;"
+                              target="_blank">View candidatures</a>
+                          </div>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="background: #F9FAFB;" width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" style="padding: 28px 56px;" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 16px; line-height: 24px; color: #A7ADB5;">This email was autogenerated by MyAEGEE mailer.</div>
```

Comment:
For a future PR; might be nice to reference the helpdesk here for if people have questions